content/en/dashboards/guide/wildcard_examples.md

```diff
@@ -0,0 +1,641 @@
+         ]
+       }
+     },
+     {
+       "field": "max",
+       "formatType": "hoverFormatter",
+       "format": {
+         "units": [
+           "nanosecond",
+           null
+         ]
+       }
+     },
+     {
+       "field": "relative_frequency",
+       "format": "0.3f"
+     }
+   ]
+ },
+ "layer": [
+   {
+     "data": {
+       "name": "table2"
+     },
+     "transform": [
+       {
+         "joinaggregate": [
+           {
+             "op": "sum",
+             "field": "count",
+             "as": "total_count"
+           }
+         ]
+       },
+       {
+         "calculate": "datum.count / datum.total_count",
+         "as": "relative_frequency"
+       }
+     ],
+     "mark": {
+       "type": "rect",
+       "color": "gray",
+       "opacity": 0.8
+     }
+   },
+   {
+     "data": {
+       "name": "table1"
+     },
+     "transform": [
+       {
+         "joinaggregate": [
+           {
+             "op": "sum",
+             "field": "count",
+             "as": "total_count"
+           }
+         ]
+       },
+       {
+         "calculate": "datum.count / datum.total_count",
+         "as": "relative_frequency"
+       }
+     ],
+     "mark": {
+       "type": "rect",
+       "color": "pink",
+       "tooltip": {
+         "content": "data"
+       }
+     }
+   }
+ ]
+}
+
+```
+{{% /collapse-content %}}
+
+## Text color scatterplot
+
+Create a customized scatterplot that uses text marks instead of points, with automatic coloring from the Datadog palette. This improves readability by displaying the actual text labels directly on the chart while maintaining the positional data relationships of a traditional scatterplot.
+
+**Key features**:
+- Text elements as data points for enhanced readability
+- Automatic color assignment using the Datadog color scheme
+- Customizable axes for different metrics (p50/p95 in this example)
+- Visual encoding that combines position and color for multi-dimensional analysis
+- Compact representation of categorical data in a quantitative space
+
+{{< img src="dashboards/guide/wildcard_examples/text_color_scatterplot.png" alt="Text-based scatterplot using the Datadog color palette to visualize p50 vs p95 metrics" style="width:100%;" >}}
+
+{{% collapse-content title="Vega-Lite Configuration" level="h4" expanded=false %}}
+```javascript
+{
+ "$schema": "https://vega.github.io/schema/vega-lite/v5.json",
+ "data": {
+   "name": "table1"
+ },
+ "encoding": {
+   "x": {
+     "field": "query1",
+     "type": "quantitative",
+     "axis": {
+       "title": "p50"
+     }
+   },
+   "y": {
+     "field": "query2",
+     "type": "quantitative",
+     "axis": {
+       "title": "p95"
+     }
+   },
+   "text": {
+     "field": "viz"
+   },
+   "color": {
+     "field": "viz",
+     "type": "nominal",
+     "scale": {
+       "scheme": "dogcat"
+     }
+   }
+ },
+ "mark": {
+   "type": "text"
+ }
+}
+
+```
+{{% /collapse-content %}}
+
+## Categorical Heatmap with Time Filtering
```

Comment:
noticed that there are two titles for this one!

Reply:
```suggestion
```